example/lib/pages/picture_in_picture_page.dart

```diff
@@ -86,6 +123,20 @@ class _PictureInPicturePageState extends State<PictureInPicturePage> {
               _betterPlayerController.disablePictureInPicture();
             },
           ),
+          // Button for testing.
+          ElevatedButton(
+            child: Text('Auto PIP: ' + (_shouldStartPIP ? 'ON' : 'OFF')),
+            onPressed: () async {
+              setState(() {
+                if (Platform.isAndroid) {
+                  _shouldStartPIP = !_shouldStartPIP;
+                }
+                _betterPlayerController
+                    .setupAutomaticPictureInPictureTransition(
+                        willStartPIP: _shouldStartPIP);
```

Comment:
Because the `PictureInPicture` function on Android is set on `activity` class.
```
<activity
            android:name=".MainActivity"
            android:configChanges=
                "screenSize|smallestScreenSize|screenLayout|orientation"
...
            android:resizeableActivity="true"
            android:supportsPictureInPicture="true"
```

 So when closing player view and back to list screen, app is still in `PictureInPicture` mode when going to the background.  So we should add this statement to `dispose` function too.

```
_betterPlayerController.setupAutomaticPictureInPictureTransition(willStartPIP: false);
```

Reply:
Used WillPopScope instead, because there was exception when do it in dispose. 
https://github.com/dwango-nfc/betterplayer/pull/13/commits/b1801cbc50f5a98aab4e2ec41896019d98b54f78

Reply:
I understood. Thank you.